imagenet/main.py

```diff
@@ -186,17 +186,15 @@ def train(train_loader, model, criterion, optimizer, epoch):
         # measure data loading time
         data_time.update(time.time() - end)
 
-        target = target.cuda(async=True)
-        input_var = torch.autograd.Variable(input)
-        target_var = torch.autograd.Variable(target)
+        target = target.cuda(True)
 
         # compute output
-        output = model(input_var)
-        loss = criterion(output, target_var)
+        output = model(input)
+        loss = criterion(output, target)
 
         # measure accuracy and record loss
-        prec1, prec5 = accuracy(output.data, target, topk=(1, 5))
-        losses.update(loss.data[0], input.size(0))
+        prec1, prec5 = accuracy(output, target, topk=(1, 5))
+        losses.update(loss.item() input.size(0))
```

Comment:
^ you also lost the comma between the arguments above?

Reply:
thanks for the catch, fixed via https://github.com/pytorch/examples/commit/56114a62cd9f35e23de2858226eb47fbc11f13de